spec/spec_helper.rb

```diff
@@ -25,9 +25,7 @@
   GC.auto_compact = true
 end
 
-def java?
-  /java/ =~ RUBY_PLATFORM
-end
+IS_JRUBY = Kernel.const_defined?(:JRUBY_VERSION)
```

Comment:
```suggestion
IS_JRUBY = RUBY_ENGINE == 'jruby'
```

Reply:
Thanks, I actually moved this to https://github.com/msgpack/msgpack-ruby/pull/251, as it's unrelated to this PR.